languages/csharp/exercises/concept/casting/.meta/Example.cs

```diff
@@ -15,7 +14,7 @@ public string GetDisplayName(TeamSupport support)
         }
         else
         {
-            return "Too Important for a Laminate";
+            return "Too Important for a Security Pass";
```

Comment:
One final nit: the tests also need to be updated.

Reply:
Done